src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -243,11 +243,18 @@ def fall_detect(self, image=None):
                 pose_dix = {}
                 inference_result = []
 
-                for e_corr in self.fall_detect_corr:
```

Comment:
@bhavikapanara As you pointed out, this filter should be restored to the extent that we want each of the hip & shoulder key points to be above the `confidence_threshold` (default value 0.6 now, not 0.25).

Reply:
Actually, there is a more succinct way to accomplish this. See the comment below for updating the formula for `right_vector_score` and `left_vector_score`.